libc-test/build.rs

```diff
@@ -1253,6 +1253,9 @@ fn test_solarish(target: &str) {
             // https://github.com/gnzlbg/ctest/issues/68
             "lio_listio" => true,
 
+            // Exists on illumos too but, for now, is a recent addition.
```

Comment:
This comment won't age well.  Perhaps include the bug number for its introduction (17094), the date of its introduction (2025-01-29), or just link to https://www.illumos.org/issues/17094 ?